puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman_packaging_release.groovy

```diff
@@ -13,10 +13,13 @@ pipeline {
     stages {
         stage('Clone Packaging') {
             steps {
+                script {
+                    $foreman_branch = foreman_version == 'nightly' ? "rpm/develop" : "rpm/${foreman_version}"
```

Comment:
You could argue we could just set this in `pipelines/vars/foreman/{version}.groovy`, but I'm fine either way.

Reply:
I did debate that. I suppose I could call it foreman_packaging_branch but given it was localized to this pipeline I figured I'd keep it close to home.